### DISCUSSION TITLE
Get feet yaw angle

### DISCUSSION THREAD

**Discussion by BlackYuzia** [saul/demofile#269] [Questions] [answered]
#### Question 

How I can get feet yaw angle or this feature is not exist in current package?

**Comment by master117** [comment]
Can't you calculate it from the last 2 positions?

**Comment by saul** [comment]
I _think_ you can get the feet yaw with `player.getProp("DT_BaseEntity", "m_angRotation")`. Can you let me know if that works?

**Comment by BlackYuzia** [comment]
@saul This isn't work. I get `undefined`

**Comment by saul** [comment]
How about `player.getProp('DT_CSPlayer', 'm_flLowerBodyYawTarget')`?

**Comment by BlackYuzia** [comment]
@saul did this feets angles?
This get some float value. 

But Idk did this feets angles or not. If yes - that works.

**Comment by BlackYuzia** [comment]
@saul `m_flGoalFeetYaw ` maybe something like this?

```
animstate->m_flGoalFeetYaw // found in source codes
```

**Comment by saul** [comment, marked as answer]
Yes - `m_flLowerBodyYawTarget` is the angle in degrees that the feet should point at. This is the target angle (i.e. it won't smoothly interpolate between two values).
